src/test/test-unaligned.c

```diff
@@ -159,7 +159,26 @@ static void test_le(void) {
         assert_se(memcmp(&scratch[7], &data[7], sizeof(uint64_t)) == 0);
 }
 
+static void test_ne(void) {
+        uint16_t x = 4711;
+        uint32_t y = 123456;
+        uint64_t z = 9876543210;
```

Comment:
Those are all going to be aligned, no? So the test doesn't test *unaligned* access...